geo/src/algorithm/geodesic_intermediate.rs

```diff
@@ -0,0 +1,130 @@
+use crate::{CoordFloat, Point};
+use geographiclib_rs::{DirectGeodesic, Geodesic, InverseGeodesic};
+
+/// Returns a new Point along a route between two existing points on an ellipsoidal model of the earth
+
+pub trait GeodesicIntermediate<T: CoordFloat> {
+    /// Returns a new Point along a route between two existing points on an ellipsoidal model of the earth
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # #[macro_use] extern crate approx;
```

Comment:
I could not justify these line

-    /// # #[macro_use] extern crate approx;
-    /// #

when I ran cargo doc and looked at the generated HTML

geo/target/doc/geo/algorithm/geodesic_intermediate/trait.GeodesicIntermediate.html

It did not appear in the output... perhaps we can just simplify by removing them.

Reply:
Within a doc comment code block, line starting with a `#` denote that the line should not be output into the docs, but they *are* executed (and necessary) when running the doc-test. 

It's commonly used to keep the rendered docs concise and focused on the usage, rather than implementation details like what testing library we use.